broker-cli/commands/wallet.js

```diff
@@ -130,29 +130,42 @@ async function commitBalance (args, opts, logger) {
     const { uncommittedBalance } = balances.find(({ symbol: s }) => s === symbol)
 
     const totalUncommittedBalance = Big(uncommittedBalance)
+
     if (totalUncommittedBalance.eq(0)) {
       return logger.info('Your current uncommitted balance is 0, please add funds to your daemon')
     }
-    // We try to take the `Math.min` total here between 2 Big numbers due to a
+
+    // We try to take the lowest total here between 2 Big numbers due to a
     // commit limit specified as MAX_CHANNEL_BALANCE
     let maxSupportedBalance = uncommittedBalance
 
     if (totalUncommittedBalance.gt(ENUMS.MAX_CHANNEL_BALANCE)) {
       maxSupportedBalance = ENUMS.MAX_CHANNEL_BALANCE
```

Comment:
shouldn't this be per currency?

Reply:
@treygriffith I added a TODO and have thrown an issue in our backlog.